src/Main.cs

```diff
@@ -152,6 +152,46 @@ private void LoadJoinedGuilds()
                 serversLv.Items.Add(lvItem);
             }
         }
+        private void LoadTopVC()
+        {
+            topVC.Items.Clear();
+            topVCGuilds.Items.Clear();
+
+            topVC.Items.AddRange(
+                DataPackage.VoiceDisconnections
+                .GroupBy(x => x.ChannelId)
+                .Select(
+                    x => new DVoiceConnection() { 
+                        ChannelId = x.First().ChannelId,
+                        GuildId = x.First().GuildId,
+                        Duration = new TimeSpan(x.Sum(y => y.Duration.Ticks))
+                    })
+                .OrderByDescending(x => x.Duration)
+                .Select(
+                    x => new ListViewItem(new string[] {
+                        (x.ChannelId != null && DataPackage.ChannelsMap.ContainsKey(x.ChannelId)) ?
+                        (DataPackage.ChannelsMap[x.ChannelId].GetName(DataPackage.User, DataPackage.UsersMap) ?? "") : "", // Get channel name from ChannelId
+                        x.ChannelId,
+                        x.Duration.ToString(@"%d'days '%h\h%m\m%s\s"),
```

Comment:
I think `1days 2h3m4s` looks too cramped. Maybe instead something like `1d 2h 3m 4s`?
```suggestion
                        x.Duration.ToString(@"%d'd '%h'h '%m'm '%s's '"),
```